src/frontend/boilerplates/archipelago/src/resources/Agent/Actor/Organization/OrganizationShow.js

```diff
@@ -16,23 +16,48 @@ import { MapField } from '@semapps/geo-components';
 import {
   ReferenceArrayField,
   ReferenceField,
-  GroupedArrayField,
-  FilteredArrayField
+  GroupedArrayField
 } from '@semapps/semantic-data-provider';
 import OrganizationTitle from './OrganizationTitle';
 import DescriptionIcon from '@material-ui/icons/Description';
 import HomeIcon from '@material-ui/icons/Home';
 
+import Chip from '@material-ui/core/Chip'
+
+function handleClick (item)
+{
+  window.location.href = item;
+}
+const TextArrayField = ({ record, source }) => {
+  const array = record[source]
+  if (typeof array === 'undefined' || array === null || array.length === 0) {
+    return <div/>
+  } else {
+    return (
+      <>
+        {array.map(item => <Chip label={item} key={item} onClick={handleClick.bind(this, item)} />)}
```

Comment:
Le bind n'est pas nécessaire puisque tu n'utilise pas this dans handleClick
Cela dit je suis surpris que tu sois obligé de gérer ça manuellement. Si c'est le cas, il vaudrait mieux mettre un vrai lien `<a>` car ça permet de faire un clique droit. De plus il faudrait mettre un target="_blank"

Reply:
J'ai du mal a comprendre comment récupérer le item sans le bind. Et pour le force array, il est obligatoire pour le moment car je fais un map sur le tableau des liens.

Reply:
> J'ai du mal a comprendre comment récupérer le item sans le bind. 
Le bind sert juste à garder la référence à this.  Il te suffit de mettre `onClick={() => handleClick(item)}`

> Et pour le force array, il est obligatoire pour le moment car je fais un map sur le tableau des liens.
Tu pourrais très bien faire un check `Array.isArray` au début et, si ce n'est pas un array, mettre la valeur dans un array.